optimism/src/mips/interpreter.rs

```diff
@@ -733,7 +733,26 @@ pub fn interpret_rtype<Env: InterpreterEnv>(env: &mut Env, instr: RTypeInstructi
         RTypeInstruction::SyscallWritePreimage => (),
         RTypeInstruction::SyscallWriteOther => (),
         RTypeInstruction::SyscallFcntl => (),
-        RTypeInstruction::SyscallOther => (),
+        RTypeInstruction::SyscallOther => {
+            let syscall_num = env.read_register(&Env::constant(2));
```

Comment:
```suggestion
            let syscall_num = env.read_register(&Env::constant(REGISTER_V0));
```

Depends on https://github.com/o1-labs/proof-systems/pull/1437